### ISSUE TITLE
Command to release a new version of a plugin

### ISSUE THREAD

**Issue by mathieudutour** [sketch-pm/skpm-cli#2]
something like `sketch-builder release [major|minor|patch]`:
   * `npm version ...`
   * `sketch-builder build`
   * `zip plugin`
   * `git push`
   * use github api to create release
   * upload zip

https://github.com/atom/apm/blob/master/src
https://gist.github.com/caspyin/2288960
https://github.com/aktau/github-release

**Comment by adamkirkwood** [comment]
Hey @mathieudutour,

I recently saw your post about a [Plugin registry](http://sketchplugins.com/d/17-plugin-registry), and I immediately thought I should reach out.

We recently had a similar thought for [Sketchpacks](https://www.sketchpacks.com) about creating a build tool to help Plugin Authors, but fortunately for us we stumbled upon your project!

We love the idea behind what you've done with Sketch Builder so far and I'd love to know how might [Sketchpacks](https://www.sketchpacks.com) could collaborate with your CLI tool.

Cheers!
